operator/cmd/mesh/install.go

```diff
@@ -134,29 +134,35 @@ func runApplyCmd(cmd *cobra.Command, rootArgs *rootArgs, iArgs *installArgs, log
 	var opts clioptions.ControlPlaneOptions
 	kubeClient, err := kube.NewExtendedClient(kube.BuildClientCmd(iArgs.kubeConfigPath, iArgs.context), opts.Revision)
 	if err != nil {
-		return err
+		operation := "encounter error when creating Kubernetes client, error detail"
+		return wrapErrorf(operation, err)
```

Comment:
```suggestion
		return fmt.Errorf("create Kubernetes client: %v", err)
```

This follows standard patterns in the rest of the code base. When creating errors like this, we get useful messages as errors propogate up, like `installation failed: create Kubernetes client: permission denied` or similar

Reply:
similar comment applies to the rest

Reply:
got it

Reply:
@howardjohn the code style is the same now. ^_^